model/Bom.js

```diff
@@ -69,25 +70,37 @@ class Bom extends CycloneDXObject {
 
   /**
    * Given the specified package, create a CycloneDX component and add it to the list.
+   *
+   * @returns {Component}
    */
   createComponent (pkg, list, lockfile, isRootPkg = false) {
     // read-installed with default options marks devDependencies as extraneous
     // if a package is marked as extraneous, do not include it as a component
     if (pkg.extraneous) return
-    if (!isRootPkg) {
-      const component = new Component(pkg, this.includeLicenseText, lockfile)
+    let component
+    if (isRootPkg) {
+      component = this._metadata.component
```

Comment:
the root component must be part of the `bom.depndencies`, too

Reply:
Regardless if the root component has any dependencies?

Reply:
yes.
all components and services must be part of `bom.dependencies`

https://cyclonedx.org/use-cases/#dependency-graph
> Components that do not have their own dependencies MUST be declared as empty elements within the graph. Components that are not represented in the dependency graph MAY have unknown dependencies. It is RECOMMENDED that implementations assume this to be opaque and not an indicator of a component being dependency-free.

Reply:
This should be resolved now, would you agree?

Reply:
yes